### PR TITLE
[TF-4327] Epic frame redesign

### PR DESCRIPTION
**Pull request by mechterp10**
Update the world frame from "stand" to "base". Remove references to the cart as this in now handled in ready_attachments

### CODE REVIEW

**Review by adamrr — APPROVED** [review]

File: ur_description/urdf/ur10.urdf.xacro [hunk 1]
```diff
@@ -308,8 +308,8 @@
                  degrees)
       -->
       <origin xyz="0 0 0" rpy="0 0 ${-pi}"/>
-      <parent link="${prefix}base_link"/>
-      <child link="${prefix}base"/>
+      <parent link="${prefix}base"/>
+      <child link="${prefix}base_link"/>
```
Comment (adamcalhoon) [inline ur_description/urdf/ur10.urdf.xacro:312]: Were these just wrong before? ditto for subsequent similar changes.

File: ur_description/urdf/ur3_joint_limited_robot.urdf.xacro [hunk 2]
```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0"?>
-<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
-       name="ur3" >
-
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="ur3" >
```
Comment (adamcalhoon) [inline ur_description/urdf/ur3_joint_limited_robot.urdf.xacro]: ?
```
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="READY_UR3" >
```

File: ur_description/urdf/ur3_robot.urdf.xacro [hunk 3]
```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0"?>
-<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
-       name="ur3" >
-
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="ur3" >
```
Comment (adamcalhoon) [inline ur_description/urdf/ur3_robot.urdf.xacro]: ?
```
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="READY_UR3" >
```

**Review by adamcalhoon — COMMENTED** [review]
Questions for you...

File: ur_description/urdf/ur10.urdf.xacro (hunk 1 above)
Comment (mechterp10) [reply ur_description/urdf/ur10.urdf.xacro:312]: No, but this is required to have base be the fixed frame and not base_link.

Comment (mechterp10) [reply ur_description/urdf/ur10.urdf.xacro:312]: This change has been made everywhere, across the board.

Comment (adamcalhoon) [reply ur_description/urdf/ur10.urdf.xacro:312]: Thanks for clarifying!

**Review by joeporembski — APPROVED** [review]
I reviewed and approved the PRs associated with each commit; however, I am not sure how I missed @adamcalhoon's comment.

I'm approving, but may be worth addressing the ur3 issue for maintenance purposes.

**Pull request merged**